app.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topics.controllers");
+const { handleServerError } = require("./errors/errors");
+
+const app = express();
+
+app.get("/api/topics", getTopics);
+
+app.all("*", (req, res, next) => {
```

Comment:
Awesome. Just fyi, express also handles undefined routes by default by sending a 404 status code (I wouldn't refactor, your way allows you to send a custom error message which is nice, but might be interesting to see what happens if you comment your 404 error handler out on lines 9-11!)